pkgs/development/python-modules/sqlalchemy-utils/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, six
+, sqlalchemy
+, pytest
+, flexmock
+, mock
+, python-dateutil
+, pytz
+}:
+
+buildPythonPackage rec {
+  pname = "SQLAlchemy-Utils";
+  version = "0.33.6";
+
+  src = fetchPypi {
+    inherit pname version;
```

Changing the pname will most likely make this not work here.
So we might need:
```suggestion
    pname = "SQLAlchemy-Utils";
    inherit version;
```